lms/djangoapps/ccx/api/v0/tests/test_views.py

```diff
@@ -111,8 +111,8 @@ def get_auth_token(self):
         token_resp = self.client.post('/oauth2/access_token/', data=token_data)
         self.assertEqual(token_resp.status_code, status.HTTP_200_OK)
         token_resp_json = json.loads(token_resp.content)
-        self.assertIn('access_token', token_resp_json)
-        return 'Bearer {0}'.format(token_resp_json.get('access_token'))
+        return '{token_type} {token}'.format(token_type=token_resp_json['token_type'],
+                                             token=token_resp_json['access_token'])
```

Comment:
Nit: See the formatting style for edx-platform.  Ending paren should be on its own line, have a carriage return after the beginning paren, and indent 4 spaces.